share/completions/toot.fish

```diff
@@ -0,0 +1,87 @@
+set -l _sub_commands 'login login_cli activate logout auth tui whoami whois notifications instance search thread timeline post upload delete favourite unfavourite reblog unreblog reblogged_by pin unpin follow unfollow mute unmute block unblock'
+
+function __fish_complete_toot_accounts
+    toot auth | awk '/^\*/{print $2}'
```

Comment:
This looks like a portable use of `awk` so I'm not really concerned, but fyi `string replace -rf [pattern] [replacement]` *should* be able to also handle this (quicker since no process needs to be spawned).